tests/TraceContext/W3CTestService/index.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use App\Kernel;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\Contrib\Jaeger\Exporter as JaegerExporter;
+use OpenTelemetry\Sdk\Trace\Clock;
+use OpenTelemetry\Sdk\Trace\Sampler\AlwaysOnSampler;
+use OpenTelemetry\Sdk\Trace\SamplingResult;
+use OpenTelemetry\Sdk\Trace\SpanProcessor\BatchSpanProcessor;
+use OpenTelemetry\Sdk\Trace\TracerProvider;
+use OpenTelemetry\Trace as API;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\ErrorHandler\Debug;
+use Symfony\Component\HttpFoundation\Request;
+
+(new Dotenv())->bootEnv(dirname(__DIR__) . '/.env');
+
+if ($_SERVER['APP_DEBUG']) {
+    umask(0000);
```

Comment:
I don't think it should be a problem if we leave this as `umask(0000)` for the duration of your test period.

Reply:
Yeah, this isn't needed. I removed it 👍🏾